src/common/io.py

```diff
@@ -1146,11 +1147,17 @@ def __init__(self, fname, delayed_trajectory_loading = True):
                 Name of file.
         """
         self._fname = fname
+        self._is_filestream = isinstance(fname, io.IOBase)
```

Comment:
Docstring to init should be updated with that this is possible as well.

Reply:
Fixed.